components/atom/tag/src/Actionable/index.js

```diff
@@ -0,0 +1,71 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import ActionableTagContainer from './Container'
+
+class ActionableTag extends Component {
+  static RIGHT_ICON_PLACEMENT = 'right'
+
+  get _classNames () {
+    const {className} = this.props
+    return classnames(
+      'sui-AtomTag-actionable',
+      className
+    )
+  }
```

Comment:
what do you think about using `cx` here instead of this function?
https://github.com/SUI-Components/sui-components/blob/master/components/card/product/src/index.js#L81
would look a bit more clear to me.

Reply:
It's using the same lib, I just named it classnames because the lib is called classnames instead of cx (I don't know why we import it as cx, what has cx to do with classnames?, do someone know? It looks pretty random)